modules/apps/object/object-test/src/testFunctional/tests/dependencies/ImportedObjectWithSalesforceStorageType.json

```diff
@@ -0,0 +1,178 @@
+{
+	"accountEntryRestricted": false,
+	"accountEntryRestrictedObjectFieldName": "",
+	"actions": {
```

Comment:
Let's rename the json file: `ImportedCustomObjectWithSalesforceStorageType.json`

Since we have custom objects and standard objects, we need to be able to distinguish between them.